test/test_dagnav.py

```diff
@@ -743,3 +851,137 @@ def test_surface_load_file(request):
     # Non-STL file should not be allowed
     with pytest.raises(ValueError):
         model.create_surface(filename='badgers.exe')
+
+def test_surface_sense_runtime_error():
+    """Test senses returns default when tag is missing."""
+    model = pydagmc.Model()
+    surf = model.create_surface(global_id=1)
+    try:
+        model.mb.tag_delete(model.surf_sense_tag, (surf.handle,))
+    except RuntimeError:
+        pass
+    assert surf.senses == [None, None]
+
+def test_surface_set_sense_with_none():
+    """
+    Test setting surface sense when one or both volumes are None.
+    Also verifies the persistence of parent-child relationships according
+    to the current implementation (setter only adds, never removes).
```

Comment:
It really should remove, but it doesn't seem that this is currently possible with PyMOAB. I'll remedy that.

Reply:
Proof of effort: https://bitbucket.org/fathomteam/moab/pull-requests/735